test/command_line/helper/command_runner.rb

```diff
@@ -103,8 +103,9 @@ def grndb(command, *arguments)
   end
 
   def find_program(name, options={})
+    name = name + RbConfig::CONFIG["EXEEXT"]
```

Comment:
Ah, sorry.
`name += RbConfig::CONFIG["EXEEXT"]` is better.

Reply:
I've fixed. Thanks.